packages/components/src/_styles/spacing.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Implements [a consistent spacing system for wordpress](https://make.wordpress.org/design/2019/10/31/proposal-a-consistent-spacing-system-for-wordpress/)